python/sdk/tests/test_build.py

```diff
@@ -112,7 +112,7 @@ def test_list_tests(self, unwrap):
     def test_update_test(self, unwrap):
         updated_name = 'updated_test'
         res = unwrap(self.build.update_test)(
-            self.build,test_id=pytest.test_id, name=updated_name, technique='T1234.001', crowdstrike_expected_outcome=EDRResponse.PREVENT)
+            self.build, test_id=pytest.test_id, crowdstrike_expected_outcome=EDRResponse.PREVENT, name=updated_name, technique='T1234.001')
```

Comment:
I'm getting concerned about you!
```suggestion
            self.build, crowdstrike_expected_outcome=EDRResponse.PREVENT, name=updated_name, technique='T1234.001', test_id=pytest.test_id)
```

Reply:
ahhh ok that makes sense. I'm cool either way you prefer it